Queueomatic/Server/Endpoints/Participant/Delete/DeleteParticipantEndpoint.cs

```diff
@@ -1,27 +1,26 @@
 ﻿using FastEndpoints;
+using Queueomatic.Server.Services.ParticipantService;
 
 namespace Queueomatic.Server.Endpoints.Participant.Delete;
 
-public class DeleteParticipantEndpoint: Endpoint<DeleteParticipantRequest, DeleteParticipantResponse>
+public class DeleteParticipantEndpoint: Endpoint<DeleteParticipantRequest>
 {
+    private readonly IParticipantService _participantService;
+
+    public DeleteParticipantEndpoint(IParticipantService participantService)
+    {
+        _participantService = participantService;
+    }
+
     public override void Configure()
     {
-        Verbs(Http.DELETE);
-        Routes("/participants/{id}");
-        AllowAnonymous();
+        Delete("/participants/{id}");
+        Roles("Participant", "User");
     }
 
     public override async Task HandleAsync(DeleteParticipantRequest req, CancellationToken ct)
     {
-        try
-        {
-            var response = new DeleteParticipantResponse();
-            await SendAsync(response, 204, cancellation: ct);
-        }
-        catch (TaskCanceledException exception)
-            when(exception.CancellationToken == ct)
-        {
-            Logger.LogInformation($"Task {nameof(DeleteParticipantEndpoint)} was cancelled.");
-        }   
+        await _participantService.DeleteOneAsync(req.Id);
```

Comment:
What happens if a participant enters someone else's id? Perhaps it would be better to also send the participants id from the claim aswell check if it matches the query id. If a user is deleting someone you could check the claimsprincipal and completely disregard the claim and just delete the participant.